app/importers/trade_lead/mca_data.rb

```diff
@@ -0,0 +1,49 @@
+require 'open-uri'
+
+module TradeLead
+  class McaData
+    include Importer
+    ENDPOINT = 'http://www.dgmarket.com/tenders/RssFeedAction.do?locationISO=&keywords=Millennium+Challenge+Account&sub=&noticeType=gpn%2cpp%2cspn%2crfc&language'
+    FUNDING_SOURCE = 'Millennium Challenge Account (MCA)'
+
+    SINGLE_VALUED_XPATHS = {
+      title:        './title',
+      url:          './link',
+      description:  './description',
+      publish_date: './pubDate',
+    }.freeze
+
+    MULTI_VALUED_XPATHS = {
+      category: './category',
+    }
+    def initialize(resource = ENDPOINT)
+      @resource = resource
+    end
+
+    def import
+      Rails.logger.info "Importing #{@resource}"
+      document = Nokogiri::XML(open(@resource))
+      leads = document.xpath('rss/channel/item').map { |item| process_entry item }.compact
+      TradeLead::Mca.index leads
+    end
+
+    private
+
+    def process_entry(item)
+      item_hash = extract_fields(item, SINGLE_VALUED_XPATHS)
+      item_hash.reverse_merge! extract_multi_valued_fields(item, MULTI_VALUED_XPATHS)
+      item_hash[:url].gsub!(/www\./, 'mcc.')
+      country = item_hash[:category].delete_at(item_hash[:category].find_index { |e| /country\// =~ e })
+      country = country.match(/country\/(\w\w)/)
+      item_hash[:country] = country[1] if country.present?
```

Comment:
Can you make sure the 2 character country code is upper-cased, and also that it is a valid ISO code. You can use the gem that's used here: https://github.com/GovWizely/webservices/blob/master/lib/importer.rb#L37